packages/core/src/memdb.ts

```diff
@@ -68,26 +88,13 @@ class MemDb extends TxProcessor {
   async findAll<T extends Doc>(_class: Ref<Class<T>>, query: DocumentQuery<T>): Promise<T[]> {
     let result: Doc[]
     if (Object.prototype.hasOwnProperty.call(query, '_id')) {
-      const docQuery = query as DocumentQuery<Doc>
-      if (docQuery._id === undefined) {
-        result = []
-      } else if (typeof docQuery._id === 'string') {
-        const obj = this.objectById.get(docQuery._id)
-        result = obj !== undefined ? [obj] : []
-      } else {
-        const ids = docQuery._id.$in ?? []
-        result = []
-        for (const id of ids) {
-          const obj = this.objectById.get(id)
-          if (obj !== undefined) result.push(obj)
-        }
-      }
+      result = this.getByIdQuery(query)
     } else {
       result = this.getObjectsByClass(_class)
     }
 
     for (const key in query) {
-      if (key === '_id') continue
+      if (key === '_id' && (query._id as any)?.$like === undefined) continue
```

Comment:
Why not allow like for _id?

Model had classes, so I could search for _id: {$like: 'class:core:%'} and will retrieve all classes from plugin?

Reply:
`like` allowed for `_id`, recheck condition please, we get objects by class and then check `_id` as any other `query` field